services/submission/src/main/java/app/coronawarn/server/services/submission/checkins/EventCheckinDataFilter.java

```diff
@@ -61,4 +68,11 @@ private boolean filterOutFutureCheckins(CheckIn checkin) {
   private boolean filterByValidSignature(CheckIn checkin) {
     return traceLocationSignatureVerifier.verify(checkin.getSignedLocation());
   }
+
+  private boolean mapsTo(Integer valueToCheck, Double target) {
+    Optional<TransmissionRiskValueMapping> foundTarget =
+        trlValueMappingProvider.getTransmissionRiskValueMapping().stream()
+            .filter(m -> m.getTransmissionRiskLevel().equals(valueToCheck)).findAny();
+    return foundTarget.isPresent() && foundTarget.get().getTransmissionRiskValue().equals(target);
```

Comment:
@EugenM-SAP / @mlenkeit - what about the ones, that are NOT `present`?

Reply:
In my view, whatever does not meet the TRL mapping value, should be filtered out. Which is what happens above.